### PR TITLE
Update troubleshooting-an-app.md

### PR DESCRIPTION
**Pull request by jonquil2002**
SC: https://app.shortcut.com/replicated/story/46378/collect-a-support-bundle-using-cli

Previews:
 - https://deploy-preview-297--replicated-docs.netlify.app/enterprise/troubleshooting-an-app#create-a-support-bundle-using-the-admin-console

 - https://deploy-preview-297--replicated-docs.netlify.app/enterprise/troubleshooting-an-app#create-a-support-bundle-using-the-cli

### CODE REVIEW

**Comment by jonquil2002** [comment]
@emosbaugh  Did I make anything technically incorrect when I brought this over from your community article? I did a lot of rewording. Also, the last two bullet points - do those commands only install the plugin or do they also generate a support bundle?


@paigecalvert  I'm not thrilled with the length of the subheadings. I could cut them down, but they'd be less precise or I could just do them as bullets. The formatting looks better but less easy to scan. Thoughts?

Thanks, folks!

File: docs/enterprise/troubleshooting-an-app.md [hunk 1]
```diff
@@ -1,22 +1,80 @@
 # Troubleshooting an Application
 
-The Replicated admin console includes a Troubleshoot page where you can generate, analyze, manage, and review remediation suggestions for troubleshooting an application.
+The Replicated admin console includes a Troubleshoot page where you can generate, analyze, manage, and review remediation suggestions for troubleshooting an application and generate a support bundle that can be shared with your vendor.
```
Comment (emosbaugh) [inline docs/enterprise/troubleshooting-an-app.md:3]: generate twice here seems redundant

File: docs/enterprise/troubleshooting-an-app.md [hunk 2]
```diff
@@ -1,22 +1,80 @@
...
+1. (Optional) Click **Download bundle** to download the collected and redacted support bundle. You can send the bundle to your vendor for assistance.
+
+## Create a Support Bundle using the CLI
+
+You can generate a support bundle using the CLI instead of the admin console. For example, if an installation fails when you are using an embedded KURL cluster to install the Replicated admin console or upload the application, the admin console may not be available.
+
+Create a support bundle using one of the following methods, depending on your environment or situation.
+
+### Create a Support Bundle Using the Default kots.io Specification
+To use the default kots.io 11 specification, run the following command to create a support bundle:
```
Comment (emosbaugh) [inline docs/enterprise/troubleshooting-an-app.md:33]:
```suggestion
To use the default kots.io specification, run the following command to create a support bundle:
```

File: docs/enterprise/troubleshooting-an-app.md [hunk 3]
```diff
@@ -1,22 +1,80 @@
...
+
+  ```
+  kubectl support-bundle https://kots.io
+  ```
+
+### Create a Support Bundle on an Air Gap Server
+If you are on an air gapped server, perform the following steps to create a support bundle:
+
+1. Run the following command to copy the default kots.io 11 specification:
```
Comment (emosbaugh) [inline docs/enterprise/troubleshooting-an-app.md:42]:
```suggestion
1. Run the following command from a computer that has access to the internet to download the default kots.io specification:
```

File: docs/enterprise/troubleshooting-an-app.md [hunk 4]
```diff
@@ -1,22 +1,80 @@
...
+
+    ```
+    curl -o spec.yaml https://kots.io -H 'User-agent:Replicated_Troubleshoot/v1beta1'
+    ```
+
+1. Run the following command to upload the specification to the server, and then use the local specification:
```
Comment (emosbaugh) [inline docs/enterprise/troubleshooting-an-app.md:48]:
```suggestion
1. Upload the spec.yaml file to the air gapped server.

1. Run the following command to create a support bundle using the uploaded spec.yaml file:
```

File: docs/enterprise/troubleshooting-an-app.md [hunk 5]
```diff
@@ -1,22 +1,80 @@
...
+  ```
+### Install the Support Bundle Plugin
+- If you do not already have the `support-bundle` kubectl plugin installed, run the following command to install the plugin:
+
+  ```
+  curl https://krew.sh/support-bundle | bash
+  ```
+
+### Install the Support Bundle Plugin using krew 5
+If you have installed krew 5, run the following command to install the `support-bundle` plugin:
```
Comment (emosbaugh) [inline docs/enterprise/troubleshooting-an-app.md:76]:
```suggestion
### Install the Support Bundle Plugin using krew
If you have installed krew, run the following command to install the `support-bundle` plugin:
```

File: docs/enterprise/troubleshooting-an-app.md [hunk 6]
```diff
@@ -1,22 +1,80 @@
...
+  kubectl support-bundle http://<server-address>:8800/api/v1/troubleshoot/<app-slug>
+  ```
+
+### Create a Support Bundle when an Application is not Installed
+If the application is not installed but the admin console is running, run the following command to create a support bundle:
+
+  ```
+  kubectl support-bundle http://<server-address>:8800/api/v1/troubleshoot
+  ```
+### Install the Support Bundle Plugin
```
Comment (emosbaugh) [inline docs/enterprise/troubleshooting-an-app.md:68]: add a newline between these two lines

File: docs/enterprise/troubleshooting-an-app.md [hunk 7]
```diff
@@ -1,22 +1,80 @@
...
+
+1. Run the following command to upload the specification to the server, and then use the local specification:
+
+    ```
+    kubectl support-bundle /path/to/spec.yaml
+    ```
+
+### Create a Support Bundle when the Admin Console and Application are Installed
+
+If the admin console is running and the application is installed, run the following command to create a support bundle:
```
Comment (emosbaugh) [inline docs/enterprise/troubleshooting-an-app.md:56]:
```suggestion
If the admin console is running and the application is installed, run the following command to create a support bundle which includes any customization specific to the application from the [support-bundle.yaml manifest file](/vendor/preflight-support-bundle-creating#creating-support-bundles):
```

File: docs/enterprise/troubleshooting-an-app.md [hunk 8]
```diff
@@ -1,22 +1,80 @@
...
+
+  ```
+  kubectl support-bundle http://<server-address>:8800/api/v1/troubleshoot/<app-slug>
+  ```
+
+### Create a Support Bundle when an Application is not Installed
+If the application is not installed but the admin console is running, run the following command to create a support bundle:
```
Comment (emosbaugh) [inline docs/enterprise/troubleshooting-an-app.md:63]:
```suggestion
If the application is not installed but the admin console is running, run the following command to create a support bundle with additional customization from the admin console:
```

File: docs/enterprise/troubleshooting-an-app.md [hunk 9]
```diff
@@ -1,22 +1,80 @@
...
+1. From the admin console, select the Troubleshoot tab.
 
-The green button will start analyzing the application.
+  ![Troubleshoot](/images/troubleshoot.png)
 
-No data leaves the cluster. The analysis works by the admin console operator executing the support bundle plugin and sending the collected bundle directly to the admin console API.
+1. Click **Analyze** to start analyzing the application.
 
-Data is never sent across the internet, or to anyone else.
+  The analysis works by the admin console operator executing the support bundle plugin and sending the collected bundle directly to the admin console API.
```
Comment (emosbaugh) [inline docs/enterprise/troubleshooting-an-app.md:13]:
```suggestion
  The admin console will execute the support bundle plugin directly. Once complete, the bundle will be available on the admin console Troubleshoot page.
```

File: docs/enterprise/troubleshooting-an-app.md (hunk 1 above)
Comment (jonquil2002) [reply docs/enterprise/troubleshooting-an-app.md:3]: Good catch. Reworked that first paragraph a bit.

**Review by emosbaugh — APPROVED** [review]

**Pull request merged**